docs/tutorial.md

```diff
@@ -0,0 +1,182 @@
+# Oria Operator Tutorial
+
+An in-depth walkthrough of building and running a oria-operator with memcached-operator.
+
+## Prerequisites
+
+- [Operator SDK](https://sdk.operatorframework.io/docs/installation/) v1.21.0 or newer
+- User authorized with `cluster-admin` permissions so that you can install an operator, CRDs and change rbac.
+- [GNU Make](https://www.gnu.org/software/make/)
+
```

Comment:
This might be a bit overkill, but I think it would be nice to add here that you have to have a Kubernetes cluster running to do this stuff on. I think it would be good to include information about [KinD](https://kind.sigs.k8s.io/) in case they want to use a throwaway cluster for testing instead of one that they already have set up.

Reply:
Updated to Create an [Kind Cluster](https://kind.sigs.k8s.io/)